public/app/plugins/datasource/azuremonitor/dataquery.cue

```diff
@@ -0,0 +1,214 @@
+							// Name of Dimension to be filtered on.
+							dimension?: string
+							// String denoting the filter operation. Supports 'eq' - equals,'ne' - not equals, 'sw' - starts with. Note that some dimensions may not support all operators.
+							operator?: string
+							// Values to match with the filter.
+							filters?: [...string]
+							// @deprecated filter is deprecated in favour of filters to support multiselect.
+							filter?: string
+						} @cuetsy(kind="interface")
+
+						#GrafanaTemplateVariableQueryType: "AppInsightsMetricNameQuery" | "AppInsightsGroupByQuery" | "SubscriptionsQuery" | "ResourceGroupsQuery" | "ResourceNamesQuery" | "MetricNamespaceQuery" | "MetricNamesQuery" | "WorkspacesQuery" | "UnknownQuery" @cuetsy(kind="type")
+						#BaseGrafanaTemplateVariableQuery: {
+							rawQuery?: string
+							...
+						} @cuetsy(kind="interface")
+						#UnknownQuery: #BaseGrafanaTemplateVariableQuery & {
+										kind: "UnknownQuery"
+						}                            @cuetsy(kind="interface")
+						#AppInsightsMetricNameQuery: #BaseGrafanaTemplateVariableQuery & {
+										kind: "AppInsightsMetricNameQuery"
+						}                         @cuetsy(kind="interface")
+						#AppInsightsGroupByQuery: #BaseGrafanaTemplateVariableQuery & {
+									kind:       "AppInsightsGroupByQuery"
+									metricName: string
+						}                    @cuetsy(kind="interface")
+						#SubscriptionsQuery: #BaseGrafanaTemplateVariableQuery & {
+									kind: "SubscriptionsQuery"
+						}                     @cuetsy(kind="interface")
+						#ResourceGroupsQuery: #BaseGrafanaTemplateVariableQuery & {
+									kind:         "ResourceGroupsQuery"
+									subscription: string
+						}                    @cuetsy(kind="interface")
+						#ResourceNamesQuery: #BaseGrafanaTemplateVariableQuery & {
+									kind:            "ResourceNamesQuery"
+									subscription:    string
+									resourceGroup:   string
+									metricNamespace: string
+						}                      @cuetsy(kind="interface")
+						#MetricNamespaceQuery: #BaseGrafanaTemplateVariableQuery & {
+							kind:             "MetricNamespaceQuery"
+							subscription:     string
+							resourceGroup:    string
+							metricNamespace?: string
+							resourceName?:    string
+						} @cuetsy(kind="interface")
+
+						// @deprecated Use MetricNamespaceQuery instead
+						#MetricDefinitionsQuery: #BaseGrafanaTemplateVariableQuery & {
+									kind:             "MetricDefinitionsQuery"
+									subscription:     string
+									resourceGroup:    string
+									metricNamespace?: string
+									resourceName?:    string
+						}                  @cuetsy(kind="interface")
+						#MetricNamesQuery: #BaseGrafanaTemplateVariableQuery & {
+									kind:            "MetricNamesQuery"
+									subscription:    string
+									resourceGroup:   string
+									resourceName:    string
+									metricNamespace: string
+						}                 @cuetsy(kind="interface")
```

Comment:
it'd be nice to have some pretiffier here, the indentation and spaces can be a bit arbitrary

Reply:
Apparently there's `make fix-cue` command you can run.

Reply:
This doesn't seem to do anything for me. Afaict this is the formatted version :/